### ISSUE TITLE
Cannot reproduce the DiffuseStyleGesture+ inference for TWH

### ISSUE THREAD

**Issue by sh-taheri** [YoungSeng/DiffuseStyleGesture#26]
Hi,
I am trying to reproduce the results by following the [instructions ](https://github.com/YoungSeng/DiffuseStyleGesture/tree/master/BEAT-TWH-main#22-twh-dataset-genea-2023) in the repo, however, the resulted animation looks wrong as the hands are not moving accordingly. 

Here is a video where the left character is the gt and the right one is the predicted animation. 
https://github.com/YoungSeng/DiffuseStyleGesture/assets/30083143/6c65366b-80ab-40a8-a75b-3033b06e7671

Do you any idea why the predicted animation is too different from the demo [here](https://github.com/YoungSeng/DiffuseStyleGesture/tree/master/BEAT-TWH-main#22-twh-dataset-genea-2023)? 

Thanks!

**Comment by YoungSeng** [comment]
Hi, thanks for the interest in this work. Have you tried our pre-trained model. Is your current batch size, learning rate and number of iterations the same as ours? It looks like it is not well trained. Maybe try to increase the number of iterations further.

**Comment by sh-taheri** [comment]
Hi, thanks for your response! 
Sorry my bad! I was only trying to use your pre-trained model, but something was accidentally changed in my source code, once I reverted everything back, I was able to reproduce the results (using TWH tst data).

**Comment by YoungSeng** [comment]
Ok thanks for your reply, very glad to hear you have resolved the issue.
